api/routes.go

```diff
@@ -34,6 +34,8 @@ func (api *API) buildHTTPRoutes() {
 	api.staticRouter.DELETE("/user", api.WithDBSession(api.validate(api.userDELETE)))
 	api.staticRouter.GET("/user/limits", api.noValidate(api.userLimitsGET))
 	api.staticRouter.GET("/user/stats", api.validate(api.userStatsGET))
+	api.staticRouter.GET("/user/addpubkey", api.validate(api.userAddPubKeyGET))
+	api.staticRouter.POST("/user/addpubkey", api.validate(api.userAddPubKeyPOST))
```

Comment:
I don't really like the endpoint but `GET /user/pubkey/add` felt wrong. Also `GET /user/pubkey` would be very confusing if it doesn't actually give you the pubkey of the user and returns a challenge instead. So, I'm open to suggestions. Keeping the current name is also fine.

Reply:
Why not `/user/pubkeys/register` ? Or just the singular?
Doesn't feel that wrong from a filesystem hierarchy standpoint.